docs/sphinx/users/comlinetools/index.txt

```diff
@@ -55,6 +56,16 @@ Currently available tools include:
         for examining the OME-XML block in an OME-TIFF file (also see
         :doc:`edit`).
 
+    domainlist
+        Displays a list of imaging domains and the supported formats
+        associated with each domain.  See :doc:`domainlist` for more
+        information.
+
+    mkfake
+        Creates a "fake" high-content screen with configurable dimensions.
+        This is useful for testing how HCS metadata is handled, without
+        requiring real image data from an acquired screen.  See :doc:`mkfake`
+        for more information.
```

Comment:
Mostly they are in the order in which the pages were added, which I guess sort of happens to coincide with how generally useful each command is.  I can certainly reorder them if desired.